ios/PromotedMetricsModule.xcodeproj/project.pbxproj

```diff
@@ -173,7 +173,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 16.2;
```

Comment:
This file is actually unused. It needs to be present for Cocoapods to recognize that there are source files in this directory.

Changing this file is harmless, up to you. :-)

Reply:
Ahh, okay.  I hit a few errors that said the `IPHONEOS_DEPLOYMENT_TARGET` was out of sync so I bulk replaced a bunch of them.